src/datagen/tsptw.jl

```diff
@@ -21,20 +21,21 @@ https://arxiv.org/abs/2006.01610
 
 Basicaly finds positions with a uniform distributions, then sets the time windows by creating a feasible tour and adding
 some randomness by using uniform distributions with gap and the length of the time windows.
+
+A seed must be specified by the user to generate a specific instance. As long as Random.seed!(seed) is called at the beginning of the function, every random-based operations with be deterministic. Caution : this is not the seed that must be specified in order to generate a same set of evaluation instances across experiment.
 """
-function fill_with_generator!(cpmodel::CPModel, gen::TsptwGenerator; seed=nothing, dist = nothing, timeWindows=nothing)
-    if !isnothing(seed)
-        Random.seed!(seed)
-    end

```

Comment:
We want evaluation instances to be the same across experiments not to be the same inside an experiment (which is totally the opposite we are trying to achieve), hence the `rng` attribute has to be defined outside the `fill_with_generator!` function.